File: src/main/java/org/shark/example/service/quartz/pojo/JobDto.java
package org.shark.example.service.quartz.pojo;

import lombok.Data;

import java.util.Date;

@Data
public class JobDto {
    private String cronExpression;
    private String jobClassName;
    private String triggerGroupName;
    private String triggerName;
    private String jobGroupName;
    private String jobName;
    private Date nextFireTime;
    private Date previousFireTime;
    private Date startTime;
    private String timeZone;
    private String status;
}

File: src/main/kotlin/org/shark/example/service/quartz/pojo/JobDto.kt
package org.shark.example.service.quartz.pojo

import java.util.Date

data class JobDto(
    var cronExpression: String? = null,
    var jobClassName: String? = null,
    var triggerGroupName: String? = null,
    var triggerName: String? = null,
    var jobGroupName: String? = null,
    var jobName: String? = null,
    var nextFireTime: Date? = null,
    var previousFireTime: Date? = null,
    var startTime: Date? = null,
    var timeZone: String? = null,
    var status: String? = null
)
